pyalgostrategypool/aaroon_crossover_v2.py

```diff
@@ -0,0 +1,68 @@
+"""
+    checkout:
+        - strategy specific docs here : https://algobulls.github.io/pyalgotrading/strategies/aroon_crossover/
+        - generalised docs in detail here : https://algobulls.github.io/pyalgotrading/strategies/common_regular_strategy/
+"""
+
+
+class AroonCrossover(StrategyBase):
+    name = 'Aroon Crossover v2'
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.time_period = self.strategy_parameters['TIME_PERIOD']
+        self.main_order_map = None
+
+    def initialize(self):
+        self.main_order_map = {}
+
+    def get_crossover_value(self, instrument):
+        hist_data = self.get_historical_data(instrument)
+        aroon_down, aroon_up = talib.AROON(hist_data['high'], hist_data['low'], timeperiod=self.time_period)
+        crossover_value = self.utils.crossover(aroon_up, aroon_down)
+        return crossover_value
+
+    def strategy_select_instruments_for_entry(self, candle, instruments_bucket):
+
+        selected_instruments, meta = [], []
```

Comment:
`selected_instruments, meta = [], []`

in some of the strategies, this is:

`instruments, meta = [], []`

please search for this everywhere and make it consistent. Everywhere, it should be
`instruments, meta = [], []`

Reply:
done